### ISSUE TITLE
Windi Analysis throws H3 Warnings when using with Nuxt 3 RC13 

### ISSUE THREAD

**Issue by AndrewBogdanovTSS** [windicss/nuxt-windicss#187]
**Describe the bug**
Warnings in terminal after upgrading to Nuxt RC 13
```
 WARN  [h3] Implicit event handler conversion is deprecated. Use eventHandler() or fromNodeMi
ddleware() to define event handlers.
     Route: /api
     Handler: async (req, res, next) => {
    if (!req.url)
      return next();
    const fullpath = req.url;
    const [path, queryString = ""] = fullpath.split("?", 2);
    const query = new URLSearchParams(queryString);
    if (path === "/report.json") {
      if (!analysisReturn || query.has("force"))
        analysisReturn = await runAnalysis(windicssOptions, analysisOptions);
      res.write(JSON.stringify(analysisReturn.result));
      return res.end();
    } else if (path === "/interpret") {
      if (!analysisReturn)
        analysisReturn = await runAnalysis(windicssOptions, analysisOptions);
      const name = query.get("name") || "";
      res.write(analysisReturn.utils.processor.interpret(name).styleSheet.build());
      return res.end();
    } else if (path === "/read") {
      const filepath = query.get("path");
      if (!filepath || !_fs.existsSync.call(void 0, filepath)) {
        res.statusCode = 404;
        return res.end();
      }
      try {
        res.write(await _fs.promises.readFile(filepath, "utf-8"));
        return res.end();
      } catch (e) {
        console.error(e);
        res.statusCode = 500;
        return res.end();
      }
    }
    res.statusCode = 404;
    res.end();
  }


 WARN  [h3] Implicit event handler conversion is deprecated. Use eventHandler() or fromNodeMi
ddleware() to define event handlers.  
     Handler: function (req, res, next) {
                let extns = [''];
                let pathname = parse(req).pathname;
                let val = req.headers['accept-encoding'] || '';
                if (gzips && val.includes('gzip')) extns.unshift(...gzips);
                if (brots && /(br|brotli)/i.test(val)) extns.unshift(...brots);
                extns.push(...extensions); // [...br, ...gz, orig, ...exts]

                if (pathname.indexOf('%') !== -1) {
                        try { pathname = decodeURIComponent(pathname) }
                        catch (err) { /* malform uri */ }
                }

                let data = lookup(pathname, extns) || isSPA && !isMatch(pathname, ignores) &&
 lookup(fallback, extns);
                if (!data) return next ? next() : isNotFound(req, res);

                if (isEtag && req.headers['if-none-match'] === data.headers['ETag']) {       
                        res.writeHead(304);
                        return res.end();
                }

                if (gzips || brots) {
                        res.setHeader('Vary', 'Accept-Encoding');
                }

                setHeaders(res, pathname, data.stats);
                send(req, res, data.abs, data.stats, data.headers);
        }

```

**To Reproduce**
1. Install Nuxt 3 RC13
2. Run pnpm dev
3. See warnings
4. Analyzer doesn't start

**Expected behavior**
No warnings in the console, the analyzer works as expected

**Additional context**
------------------------------
- Operating System: `Windows_NT`
- Node Version:     `v19.0.1`
- Nuxt Version:     `3.0.0-rc.13`
- Nitro Version:    `0.6.1`                                                                  
- Package Manager:  `pnpm@7.14.1`
- Builder:          `vite`                                                                 |t
- User Config:      `alias`, `extends`, `vite`
- Runtime Modules:  `-`
- Build Modules:    `-`                                                                      
------------------------------

**Comment by mgltech** [comment]
i have got same problems ~!

**Comment by howbizarre** [comment]
I have the same problem. It comes from analyze feature.
Set to false in nuxt.config and will stop warning you for now.

```
export default defineNuxtConfig({
    modules: ["nuxt-windicss"],
    windicss: {
        analyze: false
    }
});
```

**Comment by viandwi24** [comment]
i got same warning when upgrade 2.5.2 to 2.5.3, so im try to downgrade again to 2.5.2 and warning not show again.

**Comment by AndrewBogdanovTSS** [comment]
It's very interesting since 2.5.3 release was very minimal, it only adds watch check, I wonder how it may trigger those warnings 🤔

**Comment by AndrewBogdanovTSS** [comment]
@viandwi24 I downgraded to v.2.5.2 and got even worse error
` [unhandledRejection] Cannot read properties of undefined (reading 'push') `
I don't know why downgrade worked for you, as I noticed above - there were nothing specific added in 2.5.3 that could introduce such regression
![image](https://user-images.githubusercontent.com/2766448/201519582-3007c8de-1d26-4fdd-b3dc-e62d557a47f1.png)

**Comment by viandwi24** [comment]
i use nuxt rc13 for that.

this is my repo
https://github.com/viandwi24/nuxt3-awesome-starter

**Comment by AndrewBogdanovTSS** [comment]
@viandwi24 I checked your repo and though I don't see warnings in terminal, windi analysis is broken there as well and doesn't load in the browser.
@harlan-zw btw, when running analysis separately from main nuxt process everything works well, so it's definitely a Nitro conflict

**Comment by harlan-zw** [comment]
Thanks for the issue @AndrewBogdanovTSS. A fix is released with 2.5.4
